### PR TITLE
added dynamically created inventory html

### PR DESCRIPTION
**Pull request by BMccorm**

### CODE REVIEW

File: routes/api-routes.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = function (app) {
       minRequirement: req.body.minRequirement,
     }).then(() => {
       console.log('POST complete');
+      // res.redirect("/inventory");
```
Comment (tabouzeid) [inline routes/api-routes.js:24]: delete if not used

File: public/js/newInventory.js [hunk 2]
```diff
@@ -7,6 +7,48 @@ $(document).ready(() => {
   }).then((res) => {
     console.log(res);
 
+    // <tr>
```
Comment (tabouzeid) [inline public/js/newInventory.js:10]: delete commented code

**Review by tabouzeid — CHANGES_REQUESTED** [review]
The branch has a merge conflict, please resolve it

**Comment by BMccorm** [comment]
trying to re-upload code with a new pull request

**Review by BMccorm — COMMENTED** [review]
deleted all old code on my local copy. I'll upload that next

**Pull request closed without merging**